src/net.cpp

```diff
@@ -1476,25 +1476,28 @@ void ThreadDNSAddressSeed()
     }
 
     const vector<CDNSSeedData> &vSeeds = Params().DNSSeeds();
+    const unsigned int nMaxIPs = 256;
     int found = 0;
 
     LogPrintf("Loading addresses from DNS seeds (could take a while)\n");
 
-    BOOST_FOREACH(const CDNSSeedData &seed, vSeeds) {
+    for(const CDNSSeedData &seed : vSeeds)
+    {
         if (HaveNameProxy()) {
             AddOneShot(seed.host);
         } else {
             vector<CNetAddr> vIPs;
             vector<CAddress> vAdd;
-            if (LookupHost(seed.host.c_str(), vIPs))
+            if (LookupHost(seed.host.c_str(), vIPs, nMaxIPs))
             {
-                BOOST_FOREACH(const CNetAddr& ip, vIPs)
+                for(const CNetAddr& ip : vIPs)
                 {
-                    int nOneDay = 24*3600;
+                    ++found;
+                    const int nThreeDays = 3*24*3600;
+                    const int nFourDays = 4*24*3600;
```

Comment:
I do believe it doesn't make sense to initialize those constants inside the loop.
They get reinitialized at every loop (even if sometimes the compiler may be smart enough to optimize this out)
Consts that do not depend from loop values should be always initialized outside of it.

Reply:
Don't get mad at me but ... outside all the loops (pretty much as you did with nMaxIPs)